app/views/components/_related-navigation.html.erb

```diff
@@ -33,8 +33,9 @@
           <nav role="navigation" class="app-c-related-navigation__nav-section" aria-labelledby="related-nav-<%= section_title %>" data-module="toggle">
             <ul class="app-c-related-navigation_link-list" data-module="track-click">
               <% constructed_link_array = [] %>
+              <% section_link_limit = calculate_section_link_limit(links, max_section_length) %>
               <% links.each.with_index(1) do |link, index| %>
```

Comment:
Why does the index start with 1 rather than 0?

Reply:
@fofr So that the Google Analytics tracking we're adding in https://github.com/alphagov/government-frontend/pull/636#discussion_r158231629 starts at 1 for the first section rather than 0 (as per the [Trello story](https://trello.com/c/5KR4jsTw/161-event-tracking-on-new-related-navigation-links#comment-5a3a683195c9804e58a551f0))

Reply:
I think it was originally added for something else which has since been removed, but it will now be used in the Google Analytics tracking so that numbers start from x.1 rather than x.0 https://github.com/alphagov/government-frontend/pull/652